tsl/src/bgw_policy/reorder_api.c

```diff
@@ -42,6 +41,35 @@
 #define DEFAULT_RETRY_PERIOD                                                                       \
 	DatumGetIntervalP(DirectFunctionCall3(interval_in, CStringGetDatum("5 min"), InvalidOid, -1))
 
+#define CONFIG_KEY_HYPERTABLE_ID "hypertable_id"
```

Comment:
As pointed out in the update script, I guess this key and value is redundant with the information already in the jobs table column `hypertable_id`. 

If we want to keep hypertable info in the json, then maybe it makes more sense to have the canonical name (schema+name). That is less redundant and provides at least some human-readable info in the json, similar to `index_name`.

Reply:
storing names likes that in json has serious downsides because it would need adjustments on object renames which is circumvented by storing id

Reply:
Good point about name. Still not sure about the redundant info.